src/test/ui/consts/const-fn-in-vec.rs

```diff
@@ -0,0 +1,7 @@
+// run-fail
```

Comment:
Is it run-fail or check-fail/build-fail ?

Reply:
In this case, I'm looking specifically for the error message. Which one do you think would be the best match?

Reply:
It could be check-fail or build-fail, depends on test.
Some tests only fail in `build` mode (`cargo build`) so you could try check-fail first.